pkgs/top-level/ocaml-packages.nix

```diff
@@ -719,6 +719,8 @@ let
 
     ounit = callPackage ../development/ocaml-modules/ounit { };
 
+    ounit2 = callPackage ../development/ocaml-modules/ounit2 { };
```

Comment:
This should replace `ounit`.

Reply:
Yeah, but currently that would break a lot of packages that still depend on "oUnit". I guess it would be worth it to transition, but not sure, if it makes more sense to switch right away and patch packages or keep both versions for now and update as packages update.

Reply:
There is no reason to keep the old version of oUnit. Keeping the attribute and the ocamlfind stuff is useful though, for compatibility. See 29a98f2be1961d84024c406fd437e223fb108087

Reply:
That's great, didn't know that was possible!